bastet_js/src/JsArray.re

```diff
@@ -34,17 +34,13 @@ module Alt: ALT with type t('a) = array('a) = {
 module Apply: APPLY with type t('a) = array('a) = {
   include Functor;
   let apply = (fn_array, a) =>
-    Js.Array.reduce(
-      (acc, f) => Js.Array.concat(acc, map(f, a)),
-      [||],
-      fn_array,
-    );
+    Js.Array.reduce((acc, f) => Alt.alt(acc, map(f, a)), [||], fn_array);
 };
 
 module Monad: MONAD with type t('a) = array('a) = {
   include Array.Applicative;
   let flat_map = (x, f) =>
-    Js.Array.reduce((acc, a) => Js.Array.concat(acc, f(a)), [||], x);
+    Js.Array.reduce((acc, a) => Alt.alt(acc, f(a)), [||], x);
```

Comment:
Could you do the same for `bastet/src/Array.re` and have it reference `Alt.alt`?